Makefile

```diff
@@ -39,7 +42,7 @@ uninstall: manifests
 
 # Deploy controller in the configured Kubernetes cluster in ~/.kube/config
 deploy: manifests
-	cd config/controller && kustomize edit set image controller=${IMG}
+	cd config/controller && kustomize edit set image controller=${IMAGE}:${DEV_VERSION}
```

Comment:
i believe it should be ${REPO}:${DEV_VERSION}

Reply:
Done